[rospy.client][INFO] 2022-01-28 15:10:47,784: init_node, name[/sweeper], pid[26225]
[xmlrpc][INFO] 2022-01-28 15:10:47,792: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2022-01-28 15:10:47,793: Started XML-RPC server [http://ubuntu:40727/]
[rospy.init][INFO] 2022-01-28 15:10:47,794: ROS Slave URI: [http://ubuntu:40727/]
[rospy.impl.masterslave][INFO] 2022-01-28 15:10:47,794: _ready: http://ubuntu:40727/
[rospy.registration][INFO] 2022-01-28 15:10:47,797: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2022-01-28 15:10:47,797: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2022-01-28 15:10:47,895: registered with master
[rospy.rosout][INFO] 2022-01-28 15:10:47,896: initializing /rosout core topic
[rospy.rosout][INFO] 2022-01-28 15:10:47,902: connected to core topic /rosout
[rospy.simtime][INFO] 2022-01-28 15:10:47,905: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2022-01-28 15:10:47,971: topic[/chatter] adding connection to [http://ubuntu:45229/], count 0
[rospy.internal][INFO] 2022-01-28 15:10:48,076: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2022-01-28 15:10:49,981: signal_shutdown [signal-2]
[rospy.internal][INFO] 2022-01-28 15:10:49,992: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2022-01-28 15:10:49,993: topic[/chatter] removing connection to http://ubuntu:45229/
[rospy.impl.masterslave][INFO] 2022-01-28 15:10:49,994: signal-2
[rospy.internal][WARNING] 2022-01-28 15:10:49,996: Unknown error initiating TCP/IP socket to ubuntu:37027 (http://ubuntu:45229/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 359, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.core][INFO] 2022-01-28 15:10:50,610: signal_shutdown [atexit]
